libkbfs/disk_block_cache.go

```diff
@@ -896,35 +902,52 @@ func (cache *DiskBlockCacheLocal) evictLocked(ctx context.Context,
 		}
 	}()
 	numElements := numBlocks * evictionConsiderationFactor
-	blockID, err := cache.getRandomBlockID(numElements, cache.numBlocks)
-	if err != nil {
-		return 0, 0, err
-	}
-	rng := &util.Range{Start: blockID.Bytes(), Limit: cache.maxBlockID}
-	iter := cache.metaDb.NewIterator(rng, nil)
-	defer iter.Release()
-
 	blockIDs := make(blockIDsByTime, 0, numElements)
-
-	for i := 0; i < numElements; i++ {
-		if !iter.Next() {
-			break
+	maxPriorityForEviction := 0
+
+	// Repeatedly double the search space until there are enough evictable
+	// blocks. If we reach the whole cache size, then give up and relax which
+	// blocks can be evicted.
+	for ; blockIDs.Len() < numBlocks && maxPriorityForEviction < 2; numElements *= 2 {
```

Comment:
Since 2 is a valid priority, don't we want `maxPriorityForEviction <= 2`?  Also, these priority levels should be global constants probably, for readability.

Reply:
+1 to constant.